### ISSUE TITLE
Why we need to export so much from CatchCN

### ISSUE THREAD

**Issue by weiyuan-jiang** [GEOS-ESM/GEOSgcm_GridComp#656]
https://github.com/GEOS-ESM/GEOSgcm_GridComp/blob/49c5ca34f21c08b293ad261161451a78d2c1ae9e/GEOSagcm_GridComp/GEOSphysics_GridComp/GEOSsurface_GridComp/GEOSland_GridComp/GEOS_LandGridComp.F90#L226-L230

Since we can access the export from Catchment and catchmentCN grid comp directly, why we export so many variables here? I understand that some exports from catchment may be used by the other grid comps since it is around for a while.  But the catchCN is new and no other grid comps are using those exports. Can we remove addExport from catchCN ? @gmao-rreichle @gmao-jkolassa

**Comment by gmao-jkolassa** [comment]
I do not know a lot about what needs to be exported vs what does not need to, so apologies if my answer is uninformed. But I think that this may have been put in place, because there are plans to run Catchment-CN coupled to the atmosphere for several science projects. Eunjee Lee has been doing coupled Catchment-CN runs to look at the impact of land carbon fluxes on atmospheric carbon content (her simulations were done pre-GEOSldas, but I think there were some plans to maybe continue this work with.)Also, Melanie Follette-Cook is planning to run coupled simulation with Catchment-CN to look at the impact of fire carbon emissions on atmospheric composition.

Or am I mis-understanding and these exports are not actually needed for coupled runs?

**Comment by gmao-rreichle** [comment]
@weiyuan-jiang , @gmao-jkolassa : 
My understanding is that Land GC is just a pass-through to get exports into Surface GC.  In GEOSldas, we tend to tap directly into Catch[CN] GC where data are in tile space.  The GCM HISTORY files tend to tap into Surface GC, where data are in grid space and may also be averaged across different surface types (e.g., TSURF can be the avg of the land and landice TSURF).  
In short, I think we need the exports for the GCM when it is run with CatchCN.
